src/index.scss

```diff
@@ -1,4 +1,7 @@
+@import './style/variables.scss';
+
 body {
+  background-color: $sunglow;
```

Comment:
Not completely sold on this color scheme yet, may change once app starts developing more